CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Backstage Changelog
+
+This is a best-effort changelog where we manually collect breaking changes. It is not an exhaustive list of all changes or even features added.
+
+If you encounter issues while upgrading to a newer version, don't hesitate to reach out on Discord or open an issue!
+
+## Next Release
+
+> Collect changes for the next release below
+
+### @backstage/catalog-backend
+
+- Fixed and issue with duplicated location logs. Applying the database migrations from this fix will clear the existing migration logs. https://github.com/spotify/backstage/pull/1836
```

Comment:
`Fixed an`

Reply:
👍 , bonus points if you suggest those small fixes as a change so that it's easy to accept and commit 😅